dance/utils/wrappers.py

```diff
@@ -85,3 +89,48 @@ def wrapped_func(*args):
         return func(*new_args)
 
     return wrapped_func
+
+
+import functools
+
+
```

Comment:
A decorator that modifies a class to add functionality for working with specific modalities (`mod`) in a `mudata` object.